src/neo4j_genai/generation/prompts.py

```diff
@@ -108,17 +121,42 @@ class Text2CypherTemplate(PromptTemplate):
 {examples}
 
 Input:
-{query}
+{query_text}
 
 Do not use any properties or relationships not included in the schema.
 Do not include triple backticks ``` or any additional text except the generated Cypher statement in your response.
 
 Cypher query:
 """
-    EXPECTED_INPUTS = ["schema", "query", "examples"]
+    # EXPECTED_INPUTS = ["schema", "query_text", "examples"]
+    EXPECTED_INPUTS = ["query_text"]
 
-    def format(self, query: str, schema: str, examples: str) -> str:
-        return super().format(query=query, schema=schema, examples=examples)
+    def format(
+        self,
+        schema: Optional[str] = None,
+        examples: Optional[str] = None,
+        query_text: str = "",
+        query: Optional[str] = None,
```

Comment:
Here as well, I think we can still use the `query` name, is there a reason to rename it?

Reply:
renaming was for constancy between all the prompts.

Reply:
We agree on this one, just see comment below about try/except block.